compiler/src/Language/Bond/Codegen/Cpp/Types_h.hs

```diff
@@ -146,13 +146,13 @@ namespace std
         #{initMetadata}
     };
 
-    #{template}inline void swap(#{structName}& left, #{structName}& right)
+    #{template}inline void swap(#{qualifiedStructName}& left, #{qualifiedStructName}& right)
     {
         left.swap(right);
     }|]
       where
         template = CPP.template s
-        structName = CPP.structName s
+        qualifiedStructName = CPP.qualifiedStructName cpp s
```

Comment:
I think what you're suggesting is that we inline qualifiedStructName. I'd rather not - it's a sensible function given its neighbors, and it's long enough to be uncomfortable when used twice on one line like in swap().